src/TLuaInterpreter.cpp

```diff
@@ -6974,138 +6974,90 @@ int TLuaInterpreter::permSubstringTrigger( lua_State * L )
 
 int TLuaInterpreter::permKey( lua_State *L )
 {
-    int n = lua_gettop( L );
-    int i = 3;
-
+    quint8 argIndex = 0;
```

Comment:
C++11 adds a type for this, so we have no need to use the Qt workaround - this stuff is minor so I'll just submit a commit to your branch in the interests of saving time, let me know if you disagree!